OrbitGrpcProtos/capture.proto

```diff
@@ -128,6 +128,48 @@ message GpuJob {
   uint64 dma_fence_signaled_time_ns = 11;
 }
 
+message GpuQueueSubmission {
+  GpuQueueSubmissionMetaInfo meta_info = 1;
+  repeated GpuSubmitInfo submit_infos = 2;
+  repeated GpuDebugMarker completed_markers = 3;
+  int32 num_begin_markers = 4;
```

Comment:
Is this the total number of markers begun or the remaining open markers?

Reply:
Independent of the answer, consider adding a comment in the code to clarify.

Reply:
that is the total number of begin markers in that submission.